backends/pipeline_api.py

```diff
@@ -102,6 +103,26 @@ def update_profile_from_edx(backend, user, response, is_new, *args, **kwargs):
     )
 
 
+def check_verified_email(backend, response, details, *args, **kwargs):  # pylint: disable=unused-argument
+    """Get account information to check if email was verified for account on edX"""
```

Comment:
It's a PSA best practice to check `backend.name` if you're doing back-specific logic, so we should verify this is an edx backend.